packages/app/src/Application.ts

```diff
@@ -1,5 +1,20 @@
-import { Container } from '@pixi/display';
-import { autoDetectRenderer } from '@pixi/core';
+import { Container, IDestroyOptions } from '@pixi/display';
+import { autoDetectRenderer, Renderer, IRendererOptionsAuto, AbstractRenderer } from '@pixi/core';
+import { CanvasRenderer } from '@pixi/canvas-renderer';
```

Comment:
I still dont understand is it ok to reference types from modules without direct link and how will that affect TS apps that dont use `pixi.js-legacy` . Guess we'll find out later :) At least CanvasRenderer is not build into `pixi.js` - that one i have chacked.

Reply:
I'm not actually sure this is going to fly when we generate types. We may need a more generic Renderer interface. For building the project it is fine, because node's resolve. But because `@pixi/canvas-renderer` is not a dependency, this will create issues for users later on. @zyie, do you want to try to resolve this now or later?

Reply:
Another thought, what if we don't use CanvasRenderer as a type anywhere and just use Renderer? CanvasRenderer is a subset of Renderer functionality.

Reply:
I've removed `CanvasRenderer` and replaced it with `AbstractRenderer`.

I can look into a better solution over the next couple of days and do a separate PR to fix this and #6379